src/UserSwitch/UserSwitchWrapper.tsx

```diff
@@ -96,17 +113,61 @@ const UserSwitchWrapper: FC<Props> = ({
     return redirect(redirectPath);
   };
 
-  const goToProfile = (): void => {
+  const goToSettings = (): void => {
     redirect(profilePath);
   };
 
+  const goToLikedItems = (): void => {
+    if (likedItemsPath) {
+      redirect(likedItemsPath);
+    }
+  };
+  const goToLibraryProfile = (): void => {
+    if (libraryProfilePath) {
+      redirect(libraryProfilePath);
+    }
+  };
   // const onMemberClick = (memberId: string) => () =>
   //   switchMember({ memberId, domain });
 
-  let Actions: JSX.Element[];
+  let Actions: (JSX.Element | null)[];
 
   if (currentMember && currentMember.id) {
     Actions = [
+      <MenuItem
+        key='seeSettings'
+        onClick={goToSettings}
+        id={seeProfileButtonId}
+      >
+        <ListItemIcon>
+          <SettingsIcon fontSize='large' />
+        </ListItemIcon>
+        <Typography variant='subtitle2'>{seeProfileText}</Typography>
+      </MenuItem>,
+      hasLibraryProfile ? (
+        <MenuItem
+          key='libraryProfile'
+          onClick={goToLibraryProfile}
+          id={signOutMenuItemId}
+        >
+          <ListItemIcon>
+            <ProfileIcon fontSize='large' />
+          </ListItemIcon>
+          <Typography variant='subtitle2'>{libraryProfileText}</Typography>
+        </MenuItem>
+      ) : null,
```

Comment:
I don't think we should have `null`s in the array, instead can we just push the values conditionally ?

Reply:
@spaenleh 
I did this as I want to have it custom places, middle as an example, and also null is a valid jsx element

Reply:
@LinaYahya You could choose the place with the order in the `menuItems` array prop: 
```ts
<UserSwitchWrapper
  menuItems={
    [
      { icon: <ProfileIcon />, text: 'PublicProfile', redirection_path: '/members/1234567' },
      { icon: <LikeIcon />, text: 'Liked Items', redirection_path: '/liked' },
      // ... etc
    ]
  }
  // other props
/>

Reply:
It is true `null` is a valid JSX.Element but does it really make sens to populate an array with it when you know there will be no element there ?